### ISSUE TITLE
[Feature] Adjust navbar height to contain complete label with text enhancement

### ISSUE THREAD

**Issue by hildeor** [AtB-AS/mittatb-app#649]
## Origin
_Links to received feedback, user research or other findings._

As observed in 2 usability tests with Blindeforbundet, the navbar is difficult to use in combination with text enlargement because the labels are truncated. 

![IMG_0150](https://user-images.githubusercontent.com/8980523/98247409-e5152380-1f73-11eb-9022-06e1ed084ca1.PNG)

## Proposed solution
_A coarse description of a proposed solution, that may include wireframes and graphic design._

- Adjust navbar height so that the complete labels are readable. 
- Hide icons in navbar if text is enlarged
- Change background color of active tab (?)

**Comment by mikaelbr** [comment]
Should be solved by https://github.com/AtB-AS/mittatb-app/pull/717
